File: eNote/eNote.Model/DTOs/Korisnik.cs
using eNote.Model.DTOs;
using eNote.Model.Enums;

namespace eNote.Model
{
    public class Korisnik 
    {
        public int Id { get; set; }        
        public string KorisnickoIme { get; set; }
        public string Status { get; set; }
        public string Ime { get; set; } 
        public string Prezime { get; set; }      
        public string DatumRodjenja { get; set; } 
        public string Email { get; set; }
        public string Telefon { get; set; } 
        public string Adresa { get; set; } 
        public string Uloga { get; set; }
    }
}

File: eNote/UI/enote_desktop/lib/models/korisnik.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik {
  int? id;
  String? korisnickoIme;
  String? status;
  String? ime;
  String? prezime;
  String? datumRodjenja;
  String? email;
  String? telefon;
  int? adresaId;
  String? adresa;
  int? ulogaId;
  String? uloga;

  Korisnik();

  factory Korisnik.fromJson(Map<String, dynamic> json) =>
      _$KorisnikFromJson(json);

  Map<String, dynamic> toJson() => _$KorisnikToJson(this);
}
